Weather App/ViewModel/CurrentWeatherViewModel.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  CurrentWeatherViewModel.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 22/03/22.
+//
+
+import Foundation
+import UIKit
+import CoreLocation
+
+public class WeatherViewModel {
+    let locationName = Box("Loading..")
+    let currentWeatherDescription = Box(CurrentWeatherCompact())
+    let currentAirQuality = Box(AirQualityCompact())
+    let currentWindDescription = Box(WindCompact())
+    var forecastedData = Box([DayForecastCompact]())
+    var backGroundImage = Box(UIImage())
+    var hourforecastToday = Box([HourForecastCompact]())
+    var astroToday = Box(AstroCompact())
+    
+    private static func roundTwoDecimalLabel (value: Double) -> String {
```

Comment:
What does Label in the function name signifies here? Should we have space after function name and params?

Reply:
Added it to signify that function returns a label for the double after rounding it up,
but I thing return type of string already covers that, will remove.
will also remove the space

Reply:
Done